org.kryogenix.Pick.yml

```
@@ -0,0 +1,33 @@
+app-id: org.kryogenix.Pick
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: pick-colour-picker
+
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+
+rename-desktop-file: pick-colour-picker.desktop
+rename-appdata-file: pick-colour-picker.appdata.xml
+rename-icon: pick-colour-picker
+
+modules:
+  - name: pick-colour-picker
+    buildsystem: simple
+    build-commands:
+      - install -D pick-colour-picker-flatpak-runscript.sh /app/bin/pick-colour-picker
+      - chmod a+x /app/bin/pick-colour-picker
```

```suggestion
      - install -Dm755 pick-colour-picker-flatpak-runscript.sh /app/bin/pick-colour-picker
```

install can do that for you

---

updated as suggested for these changes.